blocks/AsyncSignal.mon

```diff
@@ -151,14 +176,15 @@ event ReceiveAsyncSignal {
 	 * @param $activation The current activation.
 	 * @param value value to be put on the wire.
 	 */
-	action $timerTriggered(Activation $activation, any $payload) {	
-		log "Timer triggered for " + $payload.toString();
+	action $timerTriggered(Activation $activation, any $payload) {
         AsyncSignal signal := <AsyncSignal>$payload;
-        Value value := new Value;
-		value.value := true;
-		value.properties := signal.params;
-        value.timestamp := $activation.timestamp;
-		$setOutput_value($activation, value);
+		if($parameters.scopeToPartition or ($activation.partition=signal.partition)) {
```

Comment:
I think it should be:
```
if ($parameters.scopeToPartition and ($activation.partition=signal.partition)) or 
    not $parameters.scopeToPartition 
{
...
}
```

Reply:
I already changed it in a previous commit:
https://github.com/SoftwareAG/analytics-builder-blocks-contrib/pull/16/commits/dfc4b0e53127b738f52823e881e3770194e6cfee